config/settings/base.py

```diff
@@ -348,6 +348,12 @@ def env(name, default=NoDefaultValue, type_=str):
 }
 
 
+# SF client settings:
+SF_CALLBACK_URL = env("SF_CALLBACK_URL", default=None)
+SF_CLIENT_KEY = env("SF_CLIENT_KEY", default=None)
+SF_CLIENT_ID = env("SF_CALLBACK_URL", default=None)
+SF_CLIENT_SECRET = env("SF_CLIENT_SECRET", default=None)
```

Comment:
So these are hardcoded values from the env? It seems like they should come from the active SF connection of the current user, not the DB or env. That is, aren't they ideally unique to whatever SF org the user authenticated with?

Reply:
No, these are the parameters for the connected app which is global for the MS instance. A single installation of MetaShare uses a single connected app to identify itself to the Salesforce platform as it logs in as multiple users.